README.md

```diff
@@ -0,0 +1,61 @@
+# React capstone project - Companies
+Companies is a a React and Redux SPA application that dispalys interesting information about various companies in the world. It fetchs all the information that it displays using API request to a server.
+
+# Built With
+- React
+- Redux
+- CSS
+
+# Dev Dependencies
+- React-routerDOM
+- Redux-toolkit
+- React-test-renderer
+- createAsyncThunk
+
+# Getting Started
+  Follow the instructions below to get started.
+- Cloning the repo
+- cd into the directory
+- installing the npm packages (npm install)
+- And finally starting the app. (that will be npm start) 
+
+# Run tests
+  Type this command into the terminal:
+- npm run test
+
+# Video Link :)
+
+[Video Link](https://www.loom.com/share/7ec36e12e80341e295598cdb901ae8d2)
+
+
+# Live Demo :)
+
+[Live Demo Link](https://saeqasultani.github.io/capstone-project-/)
```

Comment:
- [ ] Kindly provide the correct link to your deployed app for this project. This link is for your HTML capstone project.

  ![der](https://user-images.githubusercontent.com/36542986/197738604-731dbb94-64b5-47d8-932c-2a58a6e11505.jpg)